### PR TITLE
Mil 46 jest react testing lib

### PR DESCRIPTION
**Pull request by evelkaw**

### CODE REVIEW

**Review by matizaj — APPROVED** [review]
ok

File: frontend/src/App.test.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {render, fireEvent, waitFor, screen} from '@testing-library/react'
```
Comment (pawelkonior) [inline frontend/src/App.test.jsx:2]: remove unused imports

File: frontend/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
+/*
```
Comment (pawelkonior) [inline frontend/jest.config.js:1]: remove all comments

**Review by pawelkonior — APPROVED** [review]
Good job!, small changes required before merge

**Review by pawelkonior — APPROVED** [review]

**Pull request merged**